janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -157,9 +157,9 @@ public static Mode parse(String mode) {
      * TODO Rename ZOOKEEPER_URL and "zookeeper-url" to ZOOKEEPER_URLS and
      * "zookeeper-urls" in future major releases.
      */
-    public static final ConfigOption<String[]> ZOOKEEPER_URL = new ConfigOption<String[]>(SOLR_NS,"zookeeper-url",
-            "URL of the Zookeeper instance coordinating the SolrCloud cluster",
-            ConfigOption.Type.MASKABLE, new String[] { "localhost:2181" });
+    public static final ConfigOption<String[]> ZOOKEEPER_URL = new ConfigOption<>(SOLR_NS, "zookeeper-url",
+        "URL of the Zookeeper instance coordinating the SolrCloud cluster",
```

Comment:
Can you revert this and the below indentation changes? It seems this indentation was by intention everywhere for CofigOptions. I would suggest fixing all or none.

Reply:
Good catch! Fixed